bigtable/use-cases/fraudDetection/README.md

```diff
@@ -0,0 +1,87 @@
+    
+3.  **Managed service:** Cloud Bigtable provides the speed and scale all in a managed service. There are also maintenance features like seamless scaling and replication as well as integrations with popular big data tools like Hadoop, Dataflow and Dataproc.
+
+## System design
+![Fraud detection design](fraud-detection-design.svg)
+
+**1.  Input/Output Cloud Pub/Sub topics:** The real-time transactions arrive at the Cloud Pub/Sub input topic, and the output is sent to Cloud Pub/Sub output topic.
+    
+**2.  ML Model:** The component that decides the probability of a transaction of being fraudulent. This project provides a pre-trained ML model and hosts it on VertexAI (See ML Model section).
+    
+**3.  Cloud Bigtable as a Feature Store:** Cloud Bigtable stores Customers’ demographics and historical data. The Dataflow pipeline queries Cloud Bigtable in real-time when to get and aggregate the customer’s demographics and historical data.
+    
+**4.  Dataflow Pipeline:** The streaming pipeline that orchestrates this whole operation. It reads the transaction details from the Cloud Pub/Sub input topic, queries Cloud Bigtable to build a feature vector that is sent to the ML model, and lastly, it writes the output to the Cloud Pub/Sub output topic.
+    
+**5.  Data-warehouse (BigQuery, Spark, etc):** This component stores the full history of all transactions queried by the system. It runs batch jobs for continuously training the ML model. Note that this component is outside the scope of this project as a pre-trained ML model is provided for simplicity.
+
+The system design is written using the Terraform framework. All components details can be found in the file **terraform/main.tf** and it includes the components listed above.
+
+## Datasets
+
+This project uses [Sparkov Data Generation Simulator](https://github.com/namebrandon/Sparkov_Data_Generation) to generate the datasets that are used for training the ML model and for testing it.
+
+The directory **terraform/datasets/training_data** stores the datasets used for training the ML model. A pre-trained ML model comes with this project, but a custom ML model can be trained as well.
+
+The directory **terraform/datasets/testing_data** stores the datasets that can be used for testing the ML model. The ML model was never trained against these transactions. Two testing datasets are provided: a dataset containing fraudulent transactions, and another dataset containing legitimate transactions.
+
+
+## Cloud Bigtable
+
+### Schema design
+
+In Cloud Bigtable the data is organized in tables, which have rows identified by a unique row key, they can have an arbitrary number of columns and for every row key and column multiple versions of value might be stored. This allows storing historical data within the same table row which greatly simplifies query patterns. In this application we’re going to demonstrate how easy it is to store data in Cloud Bigtable for fraud detection use-cases.
```

Comment:
Just an edit on this based on how we talk about Bigtable in the docs:

Cloud Bigtable stores data in tables, each of which is a sorted key/value map. The table is composed of rows, each of which typically describes a single entity, and columns, which contain individual values for each row. Each row/column intersection can contain multiple cells. Each cell contains a unique timestamped version of the data for that row and column.